proto/cel/expr/conformance/test/simple.proto

```diff
@@ -141,3 +145,107 @@ message UnknownSetMatcher {
   // Success if we match any of these sets.
   repeated cel.expr.UnknownSet unknowns = 1;
 }
+
+// A test suite is a collection of tests designed to evaluate the correctness of
+// a CEL policy, a CEL expression or the conformance of a CEL implementation to
+// the standard specification.
+message TestSuite {
+  // The name of the test suite.
+  string name = 1;
+
+  // Description of the test suite.
+  string description = 2;
+
+  // Test sections of the test suite.
+  // Each section represents a behavior to be tested.
+  repeated TestSection sections = 3;
+}
+
+// A collection of related test cases.
+message TestSection {
+  // Name of the test section.
+  string name = 1;
+
+  // Description of the test section.
+  string description = 2;
+
+  // Test cases of the test section.
+  // Each test case represents a test scenario.
+  repeated TestCase tests = 3;
+}
+
+// A test to validate a CEL policy or expression. The test case encompasses
+// evaluation of the compiled expression using the provided input bindings and
+// asserting the result against the expected result.
+// It can also validate a raw CEL expression string through parse, check and
+// eval stages, making use of the augmenting CEL environment if provided.
+message TestCase {
+  // Name of the test case.
+  string name = 1;
+
+  // A description of the test.
+  string description = 2;
+
+  // The text of the CEL expression.
+  string expr = 3;
+
+  // Serialized environment to be used for compilation and evaluation of the
+  // CEL expression for the current test case.
+  // This option allows validating the same expression against multiple
+  // environments.
+  cel.expr.conformance.Environment env = 4;
+
+  // Input for the test case
+  oneof input_kind {
+    // A set of variable bindings to be used for evaluating a checked
+    // expression.
+    Bindings bindings = 5;
+
+    // A context message represents an input kind in the form of a proto
+    // message whose type is defined at runtime.
+    google.protobuf.Any context_message = 6;
+
+    // A context expression representing a context proto variable. The
+    // fields of the input proto.Messages are used as top-level variables within
+    // an Activation. The expression is evaluated using the cel environment
+    // configured for the test suite.
+    string context_expr = 7;
+  }
+
+  // Expected result of the test case.
+  oneof result_kind {
+    // A normal value, which must match the evaluation result exactly via value
+    // equality semantics. This coincides with proto equality, except for:
+    // *   maps are order-agnostic
+    // *   a floating point NaN should match any NaN
+    cel.expr.Value value = 8;
```

Comment:
Maybe we should call this `result_value` just to make the relationship with `result_expr` clearer